### PR TITLE
Tagging the tasks

### PR DESCRIPTION
**Pull request by gzukowski**
Clickup task:
https://app.clickup.com/t/8693be6ue

What:
Added the source of the issue and changed the way of handling the tags.

Why:
There was a need for adding tags, which represents the source of the issue.

### CODE REVIEW

**Comment by gzukowski** [comment]
There is a need for a token that has permission for adding tags (current devops account's permissions are too low).

File: js/clickup-add-issues/clickupClient.ts [hunk 1]
```diff
@@ -3,12 +3,24 @@ import { request } from "https";
...
     name: string;
     description: string;
+    source: string;
     tags: Array<string>;
 }
 
+
+const tagsMap: stringToValue = {
+    "transformhub": [],
+    "cloud-platform-panel" : ["front", "panel"],
```
Comment (alicja-gruzdz) [inline js/clickup-add-issues/clickupClient.ts:21]: The mapping is good, but could we move it outside the code, to a configuration file?
Then, if someone wants to modify it, they don't need to build the Sequence anew and we could also showcase it as a general code in our Samples.

**Review by alicja-gruzdz — COMMENTED** [review]
It is looking good, one comment about where to store the tags.

**Review by tomekcrm — APPROVED** [review]
lgtm

**Pull request merged**